gradle.properties

```diff
@@ -21,3 +21,5 @@ org.gradle.configureondemand=true
 
 # Required by Robolectric 4.x
 android.enableUnitTestBinaryResources=true
+android.useAndroidX=true
```

Comment:
I would probably add a space before this property so it doesn't look like it's required for robolectric.

Reply:
Good call.